### PR TITLE
Add Container Registry API

### PR DESCRIPTION
**Pull request by PhilippeVienne**
Pull request to resolve: #367 

- [x] Implement Container Registry API following documentation
- [ ] Add Documentation about API on readme
- [ ] Implement Testing for the API

### CODE REVIEW

File: src/main/java/org/gitlab4j/api/ContainerRegistryApi.java [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+     * @param olderThan       Tags to delete that are older than the given time, written in human readable form
+     *                        <code>1h</code>, <code>1d</code>, <code>1month</code>.
+     * @throws GitLabApiException if any exception occurs
+     */
+    @SuppressWarnings({"RedundantThrows", "unused"})
+    public void deleteRepositoryTagBulk(Object projectIdOrPath, Integer repositoryId, String nameRegex, Integer keepN, String olderThan) throws GitLabApiException {
+        /*
+            This function is not implemented because we need the library to support FormData on DELETE requests.
+            See: https://docs.gitlab.com/ee/api/container_registry.html#delete-repository-tags-in-bulk
+         */
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/ContainerRegistryApi.java]: It is highly unusual to have form data with the DELETE method.  Have you tried this with query params?

Comment (PhilippeVienne) [reply src/main/java/org/gitlab4j/api/ContainerRegistryApi.java]: I was just following the curl request set as example on the GitLab documentation. I will have a try with params.

Comment (gmessner) [reply src/main/java/org/gitlab4j/api/ContainerRegistryApi.java]: @PhilippeVienne 
Let me know how it goes.  If it requires form data, I'll add a ````delete()``` method that accepts form data.

Comment (PhilippeVienne) [reply src/main/java/org/gitlab4j/api/ContainerRegistryApi.java]: My local test have correctly a 202 Accepted response but the bulk DELETE seems to fail in all situation, as stated by documentation and also as you suggested with Query Params

**Comment by gmessner** [comment]
@PhilippeVienne 
I forgot to thank you for your contribution, so thank you.

I am going to go ahead and merge this PR.  Once merged, I'll look into and implement the bulk delete functionality.  I will also create the ```TestContainerRegistryApi``` test class but would like to work with you to fully implement the integration tests for this. Will provide more information later today.

**Comment by gmessner** [comment]
@PhilippeVienne 
The bulk tag delete is working correctly using query parameters, what I did find out is that if you call the bulk delete API endpoint more than once in an hour, the GitLab server simply responds with a 202 status and does absolutely nothing.  It does work on the first use though or done less than once an hour.  The “at most once an hour” is mentioned in the GitLab documentation (https://docs.gitlab.com/ee/api/container_registry.html#delete-repository-tags-in-bulk)

**Pull request merged**